src/main/java/com/jalasoft/jfc/controller/PdfConverterController.java

```diff
@@ -102,21 +111,22 @@ public String pdfConverter(
                 pdfParam.setOutputFileName(outputFileName);
                 pdfParam.setImageFormat(imageFormat);
                 pdfParam.setPagesToConvert(pagesToConvert);
+                pdfParam.setQuantityOfPage(quantityPages);
                 pdfParam.setThumbnail(thumbnail);
                 pdfParam.setWidth(width);
                 pdfParam.setScale(scale);
                 pdfParam.setHeight(height);
+                pdfParam.setRotate(rotate);
 
                 sameMd5 = "convert " + pdfConverter.convert(pdfParam).toString();
             }
-        } catch (IOException ex) {
-            ex.printStackTrace();
-        } catch (ConvertException ex) {
+        }  catch (ConvertException ex) {
             ex.printStackTrace();
         } catch (CommandValueException cve) {
             cve.printStackTrace();
-        } catch (NullPointerException nex) {
-            nex.printStackTrace();
+        } catch (Exception ex) {
+            ex.printStackTrace();
+            // response error result (400, 200)
```

Comment:
It is necessary code (line 129)?

Reply:
It is only a comment for implenting **response**

Reply:
Ok ok, don't worry